scripts/utils_test.ts

```diff
@@ -0,0 +1,169 @@
+import {
+  assertEquals,
+  assertInstanceOf,
+  assertRejects,
+} from "https://deno.land/std@0.183.0/testing/asserts.ts";
+import { stub } from "https://deno.land/std@0.183.0/testing/mock.ts";
+
+import { Counter, downloadString, regexIndexOf } from "./utils.ts";
+
+async function mockFetch({ body, init, fn }: {
+  body?: BodyInit;
+  init?: ResponseInit;
+  fn: () => void | Promise<void>;
+}): Promise<void> {
+  const fetchStub = stub(
+    globalThis,
+    "fetch",
+    () => Promise.resolve(new Response(body, init)),
+  );
+  try {
+    await fn();
+  } finally {
+    fetchStub.restore();
+  }
+}
+
+Deno.test(downloadString.name, async (t) => {
```

Comment:
I didn't think about it but like the idea (using `XXXXX.name` as a test name) 👍

Reply:
Replaced to string.

Reply:
Ah... Sorry for the confusion, but I was commenting that I like the way you write your tests.

Reply:
Got it!
Dropped these change. 😄